src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -241,6 +241,10 @@ private function getConfiguration()
             return $this->configuration = Configuration::fromNumber($mode);
         }
 
+        if (false === $mode) {
```

Comment:
`if (!$mode) {` would be even better I think.

Reply:
Fixed :)